inc/Engine/Admin/Settings/Page.php

```diff
@@ -878,20 +879,14 @@ private function assets_section() {
 	private function media_section() {
 		$lazyload_beacon  = $this->beacon->get_suggest( 'lazyload' );
 		$exclude_lazyload = $this->beacon->get_suggest( 'exclude_lazyload' );
-		$webp_beacon      = $this->beacon->get_suggest( 'webp' );
-		$dimensions       = $this->beacon->get_suggest( 'image_dimensions' );
-
-		if ( rocket_valid_key() && ! \Imagify_Partner::has_imagify_api_key() ) {
-			$imagify_link = '<a href="#imagify">';
-		} else {
-			$imagify_link = '<a href="https://wordpress.org/plugins/imagify/" target="_blank" rel="noopener noreferrer">';
-		}
+		/*$webp_beacon      = $this->beacon->get_suggest( 'webp' );*/
+		$dimensions = $this->beacon->get_suggest( 'image_dimensions' );
 
 		$this->settings->add_page_section(
 			'media',
 			[
 				'title'            => __( 'Media', 'rocket' ),
-				'menu_description' => __( 'LazyLoad, embeds, WebP', 'rocket' ),
+				'menu_description' => __( 'LazyLoad, embeds', 'rocket' ),
```

Comment:
```suggestion
				'menu_description' => __( 'LazyLoad, image dimensions', 'rocket' ),
```

Since we're removing also the embeds option, we can update the description to add the image dimensions option too which was not mentioned before

Reply:
Done